frameworks/engineering/backend.md

```diff
@@ -260,7 +260,7 @@ Check out the original backend engineering framework [here](https://eng-progress
 
     > **Examples**
     >
-    > Outbound and Inbound MasterCard clearing RFCs
+    > Outbound and Inbound MasterCard clearing proposals
```

Comment:
I wasn't sure about this line being change as I don't know if the doc name still has RFC in its name